solitude/exceptions.py

```diff
@@ -10,9 +12,17 @@
 log = getLogger('s')
 
 
+class InvalidQueryParams(ParseError):
+    status_code = 400
+    default_detail = 'Incorrect query parameters.'
+
+
 def custom_exception_handler(exc):
-    log.warning('Transaction rolled back due to error.')
-    rollback()
+    # We shouldn't be rolling back if we are inside a transaction test case
+    # until we figure out mozilla/solitude#388
+    if not settings.IN_TEST_SUITE:
+        log.warning('Transaction rolled back due to error.')
+        rollback()
```

Comment:
maybe an ``else`` block here with a log warning that transaction rollback was skipped?